### ISSUE TITLE
Feat: Add package imagemagick to list of dependencies

### ISSUE THREAD

**Issue by meiers** [algoo/preview-generator#270]
## Feature description and goals

After setting up all [mandatory dependencies](https://github.com/algoo/preview-generator#mandatory-dependencies) I could not get `ImagePreviewBuilderIMConvert` to run until I installed 

```
apt-get install imagemagick
```
on Ubuntu 20.04.

If it is a requirement it should be **mentioned in the README**, too.

Great library, thanks for your work!

**Comment by raphj** [comment]
Good catch and thanks for the kind words. Would you fancy a pull request to add imagemagick to the linked documentation?
